modules/admin_manual/nav.adoc

```diff
@@ -134,6 +134,8 @@
 *** xref:maintenance/migrating.adoc[Migrating to a Different Server]
 *** xref:maintenance/restore.adoc[Restore]
 
+** xref:oidc/index.adoc[OpenID Connect (OIDC)]
```

Comment:
Why do you put this on the same level of `configuration` and not under `configuration/server/` ?

Reply:
Because at this stage, I'm not completely sure where to put it.

Reply:
Sounds good.

Reply:
Missed in the last commit ?

Reply:
Odd, should have been.

Reply:
It was covered in the last set of changes.